src/scss/components/_card-articles.scss

```diff
@@ -26,6 +26,9 @@ article.card {
 
     box-shadow: 0px 0px 15px rgb(146, 146, 146);
     overflow: hidden;
+    &:hover{
+        box-shadow: 0px 0px 21px rgb(146, 146, 146);
```

Comment:
Schyssta bananer! FYI, när du anger 0 som värde behöver du inte skriva 0px utan bara 0. For the speeeeeed